hs/test/Main.hs

```diff
@@ -158,8 +167,13 @@ testExamplesCover p sources = testSpec label t where
       let missing = cNames \\ map takeBaseName sources
       missing `shouldBe` []
 
-testsFor :: ReachErr err =>
-  proxy err -> (FilePath -> TestTree) -> String -> FilePath -> IO TestTree
+testsFor ::
+  ReachErr err =>
+  proxy err ->
+  (FilePath -> TestTree) ->
+  String ->
+  FilePath ->
+  IO TestTree
```

Comment:
It splits long type signatures out onto one arg type per line

Reply:
Not so bad.